Assets/MRTK/Tests/PlayModeTests/BoundingBoxTests.cs

```diff
@@ -109,6 +109,67 @@ public IEnumerator BBoxOverride()
             yield return null;
         }
 
+        /// <summary>
+        /// Tests to see that the handlers grow in size when a pointer is near them
+        /// </summary>
+        [UnityTest]
+        public IEnumerator BBoxHandlerUI()
+        {
+            const int numSteps = 2;
+            var bbox = InstantiateSceneAndDefaultBbox();
+            bbox.ShowRotationHandleForX = true;
+            bbox.ShowRotationHandleForY = true;
+            bbox.ShowRotationHandleForZ = true;
+            bbox.ShowScaleHandles = true;
+            bbox.ProximityEffectActive = true;
+            bbox.ScaleHandleSize = 0.1f;
+            bbox.RotationHandleSize = 0.1f;
+            bbox.FarScale = 1.0f;
+            bbox.MediumScale = 1.5f;
+            bbox.CloseScale = 1.5f;
+            yield return null;
+            var bounds = bbox.GetComponent<BoxCollider>().bounds;
+            Assert.AreEqual(new Vector3(0, 0, 1.5f), bounds.center);
+            Assert.AreEqual(new Vector3(.5f, .5f, .5f), bounds.size);
+
+            var inputSimulationService = PlayModeTestUtilities.GetInputSimulationService();
+
+            // Definining the edge and corner handlers that will be used
+            var originalCornerHandlerScale = bbox.ScaleCornerVisuals[0].transform.localScale;
+            var cornerHandlerPosition = bbox.ScaleCorners[0].transform.position;
+            var originalEdgeHandlerScale = bbox.RotateMidpointVisuals[0].transform.localScale;
+            var edgeHandlerPosition = bbox.RotateMidpoints[0].transform.position;
+
+            // Wait for the scaling/unscaling animation to finish
+            yield return new WaitForSeconds(0.2f);
```

Comment:
if you modify the CloseGrowRate / FarGrowRate / MediumGrowRate and set them to 1.0f in test scenario you don't have to wait for the animation to finish and the scaling changes should be applied immediately

Reply:
Those variables aren't publicly modifiable so will be leaving this.